packages/gatsby-theme-carbon/src/components/ImageGallery/ImageGallery.module.scss

```diff
@@ -0,0 +1,169 @@
+$z-01: 1;
+$z-02: 2;
+
+.add-no-scroll {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  overflow: hidden;
+  margin: $spacing-07;
+}
+
+.gallery-container {
+  position: fixed;
+  margin: 0;
+  z-index: 10000;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 100%;
+  width: 100%;
+  background: rgba(0, 0, 0, 0.9);
+  overflow: hidden;
+}
+
+.gallery-grid {
+  height: 100%;
+  width: 100%;
+}
+
+.gallery-row {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  height: 100%;
+  width: 100%;
+}
+
+.nav-buttons-container {
+  width: 100%;
+  height: 100%;
+}
+
+.nav-buttons {
+  height: 100%;
+  width: 100%;
+  outline: none;
+  background: none;
+  border: none;
+  padding: 0;
+  z-index: $z-01;
+  cursor: pointer;
+}
+
+.left-nav svg {
+  position: absolute;
+  left: $spacing-09;
+}
+
+.right-nav svg {
+  position: absolute;
+  right: 1rem;
+}
+
+.first-right-nav {
+  height: 100%;
+  width: 100%;
+  right: 1rem;
+  outline: none;
+  background: none;
+  border: none;
+  padding: 0;
+  z-index: $z-01;
+  cursor: pointer;
+}
+
+.first-right-nav svg {
+  position: absolute;
+  right: $spacing-05;
+}
+
+.icon {
+  fill: $text-04;
+}
+
+.close-button {
+  display: flex;
+  position: absolute;
+  align-self: flex-end;
+  right: 0;
+  width: 14.5%;
+  height: 4rem;
+  background: none;
+  border: none;
+  padding: 0;
+  z-index: $z-02;
+  outline: none;
+  cursor: pointer;
+}
+
+.close-button svg {
+  position: absolute;
+  top: $spacing-07;
+  right: $spacing-07;
+}
+
+.nav-buttons:focus svg {
+  outline: 2px solid $focus;
+  outline-offset: -2px;
+}
+
+.close-button:focus svg {
+  outline: 2px solid $focus;
+  outline-offset: -2px;
+}
+
+@include carbon--breakpoint('md') {
+  :global(.bx--col-lg-2) {
+    flex: 0 0 16.66666667%;
+    max-width: 16.66666667%;
+  }
+
+  :global(.bx--col-lg-3) {
```

Comment:
This will redefine the rule to the class globally, we'll want to scope it to this component so we don't mess with other grids.